plugins/nfc/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "tauri-plugin-nfc"
+version = "1.0.0"
```

Comment:
is there a reason why this is not `2.0.0-alpha-0` or did you forget add a change file with `major` bump?